infra/testing/generate-variant-tests.js

```diff
@@ -8,7 +8,7 @@
 
 /**
  * @callback VariantCallback
- * @param {Object} variant
+ * @param {object} variant
```

Comment:
Hmmm, I'm pretty sure `Object` is the preferred variant here for Google JS style. Presumably upgrading `eslint-plugin-jsdoc` changed the default. Looks like it can be customized via something like:

```js
{"jsdoc":{"preferredTypes":{"object":"Object"}}}
```

https://github.com/gajus/eslint-plugin-jsdoc

I don't know if this will have any real implications, but I guess my preference would be to not have all of the type definitions change.

Reply:
Okay, fair enough. Switching back.